docs/api/touch-bar-scrubber.md

```diff
@@ -0,0 +1,23 @@
+## Class: TouchBarScrubber
+
+> Create a scrubber (a scrollablbe selector)
+
+Process: [Main](../tutorial/quick-start.md#main-process)
+
+### `new TouchBarScrubber(options)`
+
+* `options` Object
+  * `items` [ScrubberItem[]](structures/scrubber-item.md) - An array of items to place in this scruber
+  * `onSelect` Function - Called when the user taps an item that was not the last tapped item
```

Comment:
@MarshallOfSound do you think this should be `select` to match the format of `click` on buttons and `change` on color picker?